pkg/idp/idp.go

```diff
@@ -314,7 +320,7 @@ func (s *Service) GetAppConfig(ctx context.Context) (provider.AppConfig, error)
 	}, nil
 }
 
-func (s *Service) GetDefaultDexConfigSecret(name string, namespace string) *corev1.Secret {
+func GetDefaultDexConfigSecret(name string, namespace string) *corev1.Secret {
```

Comment:
There was no reason for this function to be a method and it made using it in tests more difficult